docs/changelog/fragments/113.bugfix.rst

```diff
@@ -0,0 +1,3 @@
+Fix double close issue, ddd logic to free
+the channel allocated memory within
+`__dealloc__`  - by :user:`ganeshrn`
```

Comment:
Since we use RST syntax, markdown assumptions won't work as you expect. Of course, I've set the default role to "any" but it'll only work if sphinx finds something to link against. Currently, Cython docs are not added to intersphinx so the only thing you can do is to mark it as inline code:
```suggestion
``__dealloc__`` - by :user:`ganeshrn`
```

Reply:
this should work later
```suggestion
`__dealloc__() <finalization_method>` - by :user:`ganeshrn`
```